### ISSUE TITLE
[Feature Request] Set cursor/input field to geolocation input window when user selects metadata edit button

### ISSUE THREAD

**Issue by teambvd** [pulsejet/memories#701]
**Is your feature request related to a problem? Please describe.**
This is purely a UX request, hoping to make it slightly more efficient/usable

At current when editing geotags to a photo in memories, the process is:
* (mouse) - click `edit`
* (mouse) - click inside the input field
* (keyboard) - type or paste in coordinates, then hit `enter`
* (mouse) - click `save` - as the cursor remains in input, the mouse is necessary once more, but as a photo can only be 'from' one location, this shouldn't be necessary (imo)

Using the mouse is a bit inefficient here, and while it's not a problem for a single photo, when working through hundreds (or even thousands), it can add up.

**Describe the solution you'd like**
Ideally, it'd look like:
* (mouse) - click `edit`- cursor now defaults to input field
* (keyboard) - type or paste in coordinates, then hit `enter`
* (keyboard) - hit `enter` once more to `save` (as there's only one location per photo)

**Describe alternatives you've considered**
Not sure if other solutions behave exactly this way, but at least for google's Drive, the keyboard input/shortcuts seem to be quite efficient. I don't consider google a valid alternative for me, and haven't checked/tried Photoprism / Immich / etc to verify

**Additional context**
Ever since I've had a smartphone, I've always had geotagging disabled so that this wasn't shared when backing up photos to third parties. Now that I'm self-hosting my photo backups, I'd like to edit in the geotags as I come across them, for those which I can recall at least, to make full use of the maps tie-in (amazing feature, really enjoying memories!!). 

As you can imagine, this is a significant undertaking.

**Comment by pulsejet** [comment]
Good point. This applies to not only this dialog, but probably others too.

**Comment by meonkeys** [comment]
I've also noticed that PgUp and PgDn don't always work in timeline view (in a desktop web browser)... would that fit in this issue as well?
